src/app/components/client/UpsellDialog.module.scss

```diff
@@ -1,5 +1,9 @@
 @import "../../tokens";
 
+.modal {
+  position: absolute;
+}
+
```

Comment:
What is this needed for? I'd expect this to be covered by the `<ModalOverlay>` already?

Reply:
Great catch! Rather than adding this style I just removed a redundant wrapper element in [3782e7e](https://github.com/mozilla/blurts-server/pull/3971/commits/3782e7e52d785b9d44ed6fa022f99d06bc393b25).